GVFS/GVFS.UnitTests/Service/MacServiceTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿using GVFS.Common.FileSystem;
+using GVFS.Common.Tracing;
+using GVFS.Platform.Mac;
+using GVFS.Service;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Mock.FileSystem;
+using Moq;
+using NUnit.Framework;
+using System.IO;
+
+namespace GVFS.UnitTests.Service
+{
+    [TestFixture]
+    [NonParallelizable]
+    public class MacServiceTests
+    {
+        private const string GVFSServiceName = "GVFS.Service";
+        private const string ExpectedActiveUserId = "502";
+        private static readonly string ExpectedActiveRepoPath = Path.Combine("mock:", "code", "repo2");
+        private static readonly string ServiceDataLocation = Path.Combine("mock:", "registryDataFolder");
+
+        private MockFileSystem fileSystem;
+        private MockTracer tracer;
+        private Mock<MacPlatform> gvfsPlatformMock;
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.tracer = new MockTracer();
+            this.fileSystem = new MockFileSystem(new MockDirectory(ServiceDataLocation, null, null));
+            this.gvfsPlatformMock = new Mock<MacPlatform>();
+            this.gvfsPlatformMock.Setup(p => p.GetCurrentUser()).Returns(ExpectedActiveUserId);
+            this.gvfsPlatformMock.Setup(p => p.GetUserIdFromLoginSessionId(It.IsAny<int>())).Returns<int>(x => x.ToString());
+            this.gvfsPlatformMock.SetupGet(p => p.FileSystem).Returns(new MockPlatformFileSystem());
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            this.gvfsPlatformMock = null;
+            this.fileSystem = null;
+            this.tracer = null;
+        }
+
+        [TestCase]
+        public void ServiceStartTriggersAutoMountForCurrentUser()
+        {
+            Mock<IRepoRegistry> repoRegistry = new Mock<IRepoRegistry>();
+            repoRegistry.Setup(r => r.AutoMountRepos(It.IsAny<int>()));

```

Comment:
When we know what a method is going to be called with I think it is better to specify it instead of using `It.IsAny`.
```suggestion
            repoRegistry.Setup(r => r.AutoMountRepos(ExpectedActiveUserId));
```